GeometryDashAPI/Data/LocalLevels.cs

```diff
@@ -57,8 +57,11 @@ private void RecalculateIndex()
             for (int i = 0; i < levels.Count; ++i)
             {
                 var level = levels[i];
-                if (!index.ContainsKey(level.Name))
+                if (!index.ContainsKey(level.Name)) 
                     index.Add(level.Name, new Dictionary<int, int>());
```

Comment:
You can use [GetOrCreate](https://github.com/Folleach/GeometryDashAPI/blob/dev/GeometryDashAPI/Extensions.cs#L26) instead call index method on dictionary
```
var container = index.GetOrCreate(level.Name, _ => new Dictionary<int, int>());
```